test/fixtures/wrapper/test.tf

```diff
@@ -6,8 +6,17 @@ module "vpc" {
   source = "../../../"
 
   name_prefix = "${local.name_prefix}"
+
+  price_saving_enabled = "1"
 }
 
+# @TODO Enable and test this!!!
```

Comment:
- [ ] Call VPC in non saving mode in test.f
- [ ] Test VPC non saving mode with instance in